shacl12-ui/index.html

```diff
@@ -1513,6 +1513,142 @@ <h4>Other Complex Paths</h4>
         </section>
     </section>
 
+    <section id="property-roles">
+      <h2>Property Roles</h2>
+
+      <p>
+        RDF resources commonly use properties to specify labels and descriptions, which user-interface engines depend on to render structured content.
+        However, the vocabularies defining these properties vary across domains and communities. To ensure consistent interpretation, property shapes can
+        be annotated with explicit roles for elements such as labels and descriptions.
+      </p>
+
+      <p>
+        This section introduces Property Roles for annotating SHACL property shapes along with several core property roles defined in the SHUI namespace.
+        Property Roles is defined in the SHUI namespace and defines the class <code>shui:PropertyRole</code> and the property <code>shui:propertyRole</code>.
+      </p>
+
+      <h3>Direct Role Annotation</h3>
+
+      <p>
+        Property roles can be annotated on property shapes by using the <code></code>shui:propertyRole</code> predicate and linking directly to a property role instance. 
+        SHACL renderers may use this direct annotation to drive how specific user-interface elements are displayed.
+      </p>
+
+      <p>
+        It is possible but not recommended to assign the same property role to multiple property shapes that apply to the same focus node using the direct role annotation form. 
+        When this occurs, the resulting behavior is undefined.
+      </p>
+
+      <p>
+        The example below illustrates the common case where a property shape is directly annotated with shui:LabelRole. This informs user-interfaces that the shape's value nodes
+        represent human-readable labels for resources.
+      </p>
+
+      <aside class="example" title="Direct role annotation">
+        <div class="shapes-graph">
+          <div class="turtle">
+ex:Label a sh:PropertyShape ;
+    sh:path skos:prefLabel ;
+    shui:propertyRole shui:LabelRole ;
+.
+          </div>
+        </div>
+        <p>
+          This example illustrates the common case where a property shape is directly annotated with <code>shui:LabelRole</code>. 
+          This informs user-interfaces that the shape's value nodes represent human-readable labels for resources.
+        </p>
+      </aside>
+
+      <h3>Qualified Role Annotation</h3>
```

Comment:
Should we add a note that RDF 1.2 reification is recommended, and the section will be revisited later?

Reply:
Added in https://github.com/w3c/data-shapes/pull/651/commits/7c9b1e0e07e22af79b64766c14f4d8469645b8bc